CONST_Makefile

```diff
@@ -459,6 +459,7 @@ $(OUTPUT_DIR)/build.js: .build/node_modules.timestamp .build/build.js
 		bootstrap/dist/js/bootstrap.min.js \
 		proj4/dist/proj4.js \
 		d3/d3.min.js \
+		fuse.js/src/fuse.js \
```

Comment:
This file is not minifiied. I don't think we should add non-minified file to our build.